src/main/resources/db/devonly/R__dev_data.sql

```diff
@@ -44,23 +44,33 @@ DELETE
 FROM consumer_subscription
 WHERE oauth_client_id IN ('dwp-event-receiver', 'hmrc-client', 'internal-outbound');
 
+DELETE
+FROM consumer_subscription_enrichment_field
+WHERE enrichment_field IN
```

Comment:
Could we only add and delete for the consumer subscriptions added in this. Also could we move this above the delete from consumer subscriptions